.github/workflows/build_and_publish.yml

```diff
@@ -18,23 +18,15 @@ jobs:
         with:
           python-version: "3.11"
 
-      - name: Install build tools
-        run: >-
-          python -m
-          pip install
-          wheel
-          twine
-          --user
+      - name: Install Poetry
```

Comment:
We can use a predfined workflow, for example:

```

   - name: Install poetry
     uses: abatilo/actions-poetry@v2.3.0
     with:
      poetry-version: "1.4.0"
```

Reply:
I think I prefer just installing it with `pip`